docs/docs/api/Tasks/createTasks.md

```diff
@@ -27,7 +27,8 @@ This is a **protected route**, a **valid JWT is required** in the header field
 #### Header
 
 ```
-Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
+Cookie:
+token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
 #### Body
```

Comment:
Create task no longer requires the `user_id`. It should be removed

Reply:
Since the frontend also requires a location, I added that required field (In a separate branch) but I would still include that in the documentation.

Reply:
Fixed: Changes are reflected in recent commits